src/dsp_tools/commands/xmlupload/resource_create_client.py

```diff
@@ -298,15 +297,15 @@ def _make_link_value(value: XMLValue, iri_resolver: IriResolver) -> dict[str, An
 
 def _make_list_value(value: XMLValue, iri_lookup: dict[str, str]) -> dict[str, Any]:
     s = _assert_is_string(value.value)
-    iri = iri_lookup.get(s)
-    if not iri:
+    if iri := iri_lookup.get(s):
+        return {
+            "@type": "knora-api:ListValue",
+            "knora-api:listValueAsListNode": {
+                "@id": iri,
+            },
+        }
+    else:
```

Comment:
technically, this doesn't even need the else anymore, I think

Reply:
I know, and I think that Sourcery suggested me to remove the else. But sometimes, I think that such a redundant `else` can improve legibility :-)